apple/RNCWebView.m

```diff
@@ -1559,7 +1565,7 @@ - (void)resetupScripts:(WKWebViewConfiguration *)wkWebViewConfig {
       if(!_incognito && !_cacheEnabled) {
         wkWebViewConfig.websiteDataStore = [WKWebsiteDataStore nonPersistentDataStore];
       }
-      [self syncCookiesToWebView:nil];
```

Comment:
Did you mean to remove the `syncCookiesToWebView` call here? That's not equivalent to the `addObserver` line